cvat-ui/src/reducers/index.ts

```diff
@@ -14,6 +14,15 @@ export type StringObject = {
     [index: string]: string;
 };
 
+enum AdvancedAuthMethods {
+    GOOGLE_ACCOUNT_AUTHENTICATION = 'GOOGLE_ACCOUNT_AUTHENTICATION',
```

Comment:
@Marishka17 , @bsekachev , do you think we can avoid duplication? We get the same information from the server. I will prefer to have a REST API method to get all the necessary information from the server to render UI. Now we have a list of available methods on the server and UI at the same time.